### PR TITLE
New package: lua-luadbi-0.5

### PR DESCRIPTION
**Pull request by jprjr**
Includs lua51-luadbi and lua52-luadbi

### CODE REVIEW

**Comment by jprjr** [comment]
Fixed the xlint errors (and cross-compilation), rebased+pushed.

**Comment by jprjr** [comment]
This final rebase+push should actually be working now.

File: srcpkgs/lua-luadbi/template [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+maintainer="John Regan <john@jrjrtech.com>"
+license="MIT"
+homepage="http://luadbi.googlecode.com/"
+distfiles="${homepage}files/luadbi.$version.tar.gz"
+wrksrc="${pkgname#lua-}-${version}"
+checksum=49116458dd80052536bf5d7e2b2a1f5a0fa42a671ce77f1cb7355ef7faeb331e
+create_wrksrc="yes"
+
+post_extract() {
+	cd ${wrksrc}
```
Comment (Vaelatern) [inline srcpkgs/lua-luadbi/template]: Every time you cd to a path with `${wrksrc}` is wrong, here.

**Comment by Vaelatern** [comment]
#yolo

**Pull request merged**